itemspawner/README.md

```diff
@@ -0,0 +1,19 @@
+# Item Spawner
+## Description
+Adds automated item spawners with customizable respawn time and chances of rare items dropping.
+
+## Installation
+- Download the folder (itemspawner/) or ```git clone git@github.com:NebulousCloud/helix-plugins.git```
+- Copy/move folder into your schema plugin folder
+- Add spawnpoints with /ItemSpawnerAdd
+
+## Access
```

Comment:
CAMI is a library for addons and such to register privileges with admin mods so end users can avoid having to edit Lua scripts and can customize it with a CAMI-compatible admin mod, making this section unnecessary